email_alert_service/models/email_unpublishing_processor.rb

```diff
@@ -57,7 +57,14 @@ def was_consolidated?
   end
 
   def has_alternative_url?
-    document.dig("details", "alternative_path").present?
+    present?(document.dig("details", "alternative_path"))
```

Comment:
Rather than creating a new method, can we use `blank?` instead?

Reply:
The same goes for replacing `presence`.

Reply:
Confirmed - #blank? is also missing now.

Reply:
But `empty?` is still available? I think you can use that directly.

Reply:
Okay, change of tack - perhaps just explicitly include the old monkeypatches from activesupport, since they are actually there, it turns out, just not required by other gems any more.